### ISSUE TITLE
Passphrase

### ISSUE THREAD

**Issue by streamingsystems** [kaltura/nginx-srt-module#35]
Hi,

I noticed that the default passphrase is ''. 

Does that mean that the passphrase is disabled in which case the SRT data is not encrypted? 

If so, which is fine, I am assuming then on the encoder (e.g. in OBS or Wirecast) we would just leave the passphrase blank - otherwise if we wanted to use a passphrase we would need to match the passphrase that is set in the nginx module and that that is set in the encoder (e.g. is this passphrase essentially the key used for encryption/decryption). 

thanks again!

**Comment by erankor** [comment]
Yes, by default, encryption is disabled. If you want to use encryption, you have to use the same passphrase on both ends.
